macro-fullcalendar-api/src/main/java/org/xwiki/fullcalendar/FullCalendarManager.java

```diff
@@ -38,4 +40,16 @@ public interface FullCalendarManager
      * @throws Exception in case of exceptions.
      */
     String iCalToJSON(String iCalStringURL) throws Exception;
+
+    /**
+     * Convert an iCal to a JSON.
+     *
+     * @param iCalStringURL the String representation of an iCal URL.
```

Comment:
Why not use an actual URL? This is a component, not a script service, so parameters should be typed.